disease-classification-training/unittests.py

```diff
@@ -21,13 +21,17 @@ def setUp(self) -> None:
         self.disease_classification = train_model(
             url=self.test_cfg["lr"],
             token=self.test_cfg["epochs"],
-            org=self.test_cfg["batch_size"],
-            org=self.test_cfg["model_name"]
+            bs=self.test_cfg["batch_size"],
+            model_name=self.test_cfg["model_name"]
+            train_data_size=sef.test_cfg["test_num"]
         )
 
         self.accuracy_upper_bound = self.test_cfg["accuracy_upper_bound"]
         self.accuracy_lower_bound = self.test_cfg["accuracy_lower_bound"]
-
+        self.specificity_lower_bound = self.test_cfg["specificity_lower_bound"]
+        self.specificity_upper_bound = self.test_cfg["specificity_upper_bound"]
+        self.sensitivity_lower_bound = self.test_cfg["sensitivity_lower_bound"]
+        self.sensitivity_upper_bound = self.test_cfg["sensitivity_upper_bound"]
 
```

Comment:
I do not see the config yaml in the directory?

Reply:
[test_config.yaml](https://github.com/cnvrg/disease-classification/blob/master/disease-classification-training/test_config.yaml) It's in the directory